marcel.py

```diff
@@ -49,14 +49,18 @@
     u'auboulot': u'unpause',
     u'barrage': u'wait',
     u'socialistes': u'ps',
+    u'empile': u'stack',
+    u'essaim': u'swarm',
```

Comment:
```suggestion
    u'bzzzz': u'swarm',
```

Je propose d'imiter le son des abeilles.